app/presenters/check_answers/your_details_card.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module CheckAnswers
+  class YourDetailsCard < Base
+    attr_reader :firm_details_form
+
+    KEY = 'firm_details'
+    GROUP = 'about_you'
+
+    def initialize(claim)
+      @firm_details_form = Steps::FirmDetailsForm.build(claim)
+      @group = GROUP
+      @section = KEY
+    end
+
+    def route_path
+      KEY
+    end
+
+    # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+    def rows
+      [
+        {
+          key: { text: translate_table_key(KEY, 'firm_name'), classes: 'govuk-summary-list__value-width-50' },
+          value: { text: firm_details_form.firm_office.name }
+        },
+        {
+          key: { text: translate_table_key(KEY, 'firm_account_number'), classes: 'govuk-summary-list__value-width-50' },
+          value: { text: firm_details_form.firm_office.account_number }
+        },
+        {
+          key: { text: translate_table_key(KEY, 'firm_address'), classes: 'govuk-summary-list__value-width-50' },
+          value: { text: format_address(firm_details_form.firm_office.address_line_1,
+                                        firm_details_form.firm_office.town,
+                                        firm_details_form.firm_office.postcode,
+                                        firm_details_form.firm_office.address_line_2) }
+        },
+        {
+          key: { text: translate_table_key(KEY, 'solicitor_full_name'), classes: 'govuk-summary-list__value-width-50' },
```

Comment:
Figma: 'Your full name', and we are missing ' Your email address'

Reply:
True but we are not capturing that in the defendant details page (and that is no in the defendant details page design) - I'll mention to Siddiqah

Reply:
where is this in the design? 
![image](https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/26544538/01a63c78-a9a2-437b-94f4-28ee1101baf4)

Reply:
<img width="1080" alt="Screenshot 2023-07-19 at 10 10 53" src="https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/90189839/3ef1c494-086d-430f-8beb-44a89853a4aa">
We are not capturing phone number anymore but email address

Reply:
Updated to include alternative contact full name